data/creaturescripts/scripts/reward_chest.lua

```diff
@@ -0,0 +1,304 @@
+REWARD_CHEST = {
+	BOSSES = {"bibby bloodbath", "chizzoron the distorter", "ferumbras", "furyosa", "gaz'haragoth", "ghazbaran", "hirintror", "jaul", "mad mage", "mawhawk", "morgaroth", "obujos", "ocyakao", "omrafir", "orshabaal", "raging mage", "tanjis", "the mutated pumpkin", "the pale count", "the welter", "tyrn", "white pale", "zulazza the corruptor", "zushuka"},
```

Comment:
As I said last night you can make this look alot better
BOSSES = {
      "Demon",
      "Ferumbras"
}

Etc